### PR TITLE
[JA] Translate plug 2.2.0

### PR DESCRIPTION
**Pull request by starfish719**
I translated `lessons/specifics/plug` section.
Please review! @elixirschool/japanese

### CODE REVIEW

**Comment by starfish719** [comment]
not yet WIP.
I translated to `Adding Another Plug`

File: ja/lessons/specifics/plug.md [hunk 1]
```diff
@@ -58,62 +67,85 @@ defmodule Example.HelloWorldPlug do
...
+
+この関数には3つのオプションがあります。
+
+* `:scheme` - HTTP、またはHTTPS のアトム (`:http`, `:https`)
+* `:plug` - Webサーバーのインターフェースとして使用されるプラグモジュール。
+`MyPlug`のようなモジュール名、またはモジュール名とオプション`{MyPlug, plug_opts}`のタプルを指定することができます。ここで`plug_opts`はプラグモジュールの `init/1` 関数に渡されます。
+* `:options` - サーバーオプション。
+サーバーが要求するポート番号を含める必要があります。
+
+私たちの `lib/example/application.ex` ファイルは、その `start/2` 関数で子の仕様を実装するべきです：
```
Comment (starfish719) [inline ja/lessons/specifics/plug.md]: "Our `lib/example/application.ex` file should implement the child spec in its `start/2` function:"

`child spec` is `子の仕様` ? 🤔

Comment (starfish719) [reply ja/lessons/specifics/plug.md]: not translate `child spec`
https://github.com/elixirschool/elixirschool/pull/1903/commits/4da4f3cb5efd0e00fc70d6eac29d4aefcbae662b

**Comment by starfish719** [comment]
I finished translate.
So, remove WIP.
Please review! @elixirschool/japanese

File: ja/lessons/specifics/plug.md [hunk 2]
```diff
@@ -58,62 +67,85 @@ defmodule Example.HelloWorldPlug do
...
 
-`init/1`関数はPlugのオプションを初期化するのに用いられます。そのオプションは次のセクションでご紹介するSupervisor treeから呼び出されます。ここでは、空のリストを渡します。
+`init/1`関数はPlugのオプションを初期化するのに用いられます。
+そのオプションは次のセクションでご紹介するSupervisor treeから呼び出されます。
+ここでは、空のリストを渡します。
 
 `init/1`関数の戻り値は最終的に`call/2`関数の2つ目の引数として渡されます。
 
-`call/2`関数はリクエストのたびにCowboyなどのウェブサーバーから呼び出される。`call/2`関数は`%Plug.Conn{}`構造体を第一引数として受け取り、`%Plug.Conn{}`構造体を返します。
+`call/2`関数はリクエストのたびにCowboyなどのウェブサーバーから呼び出されます。`call/2`関数は`%Plug.Conn{}`構造体を第一引数として受け取り、`%Plug.Conn{}`構造体を返します。
```
Comment (riseshia) [inline ja/lessons/specifics/plug.md]:
```suggestion
`call/2`関数はリクエストのたびにCowboyから呼び出されます。`call/2`関数は`%Plug.Conn{}`構造体を第一引数として受け取り、`%Plug.Conn{}`構造体を返します。
```

I think `Cowboy` in `from the web server, Cowboy.` is just same as web server.

File: ja/lessons/specifics/plug.md [hunk 3]
```diff
@@ -58,62 +67,85 @@ defmodule Example.HelloWorldPlug do
...
+
+この関数には3つのオプションがあります。
+
+* `:scheme` - HTTP、またはHTTPS のアトム (`:http`, `:https`)
+* `:plug` - Webサーバーのインターフェースとして使用されるPlugモジュール。
+`MyPlug`のようなモジュール名、またはモジュール名とオプション`{MyPlug, plug_opts}`のタプルを指定することができます。ここで`plug_opts`はPlugモジュールの `init/1` 関数に渡されます。
+* `:options` - サーバーオプション。
+サーバーが要求するポート番号を含める必要があります。
+
+私たちの `lib/example/application.ex` ファイルは、その `start/2` 関数でchild specを実装するべきです:
```
Comment (riseshia) [inline ja/lessons/specifics/plug.md]:
```suggestion
`lib/example/application.ex` ファイルは、その `start/2` 関数でchild specを実装するべきです:
```

**Comment by starfish719** [comment]
@riseshia Thank you for your review 👍

**Pull request merged**